cms/forms/wizards.py

```diff
@@ -294,6 +294,12 @@ def save(self, **kwargs):
                         }
                         add_plugin(**opts)
 
+        # is it home? publish it right away
+        pages_in_site = Page.objects.filter(site_id=page.site_id)
+
+        if not self.page and pages_in_site.count() == 1:
```

Comment:
I think `not pages_in_site.exclude(pk=page.pk).any()` will have better performance.